### PR TITLE
Seedlet: Fix Theme Review bugs

### PR DESCRIPTION
**Pull request by jffng**
This PR addresses the first half of [this feedback](https://themes.trac.wordpress.org/ticket/85664#comment:6). Specifically it:

- Adds missing escaping to attributes in `woocommerce.php`
- Prefixes handles in the custom colors class
- Replaces `@package WordPress` with `@package Seedlet` 

I'm adding comments in line where I'm not sure what needs to be done.

### CODE REVIEW

File: seedlet/classes/class-seedlet-custom-colors.php [hunk 1]
```diff
@@ -293,7 +292,7 @@ function on_customize_controls_enqueue_scripts() {
 		];
 
 		wp_enqueue_script( $handle, $src, $deps );
-		wp_script_add_data( $handle, 'data', sprintf( 'var _validateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
+		wp_script_add_data( $handle, 'data', sprintf( 'var seedletValidateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
 
 		// Custom color contrast validation text
 		wp_localize_script( $handle, 'validateContrastText', esc_html__( 'This color combination may be hard for people to read. Try using a brighter background color and/or a darker foreground color.', 'seedlet' ));
```
Comment (jffng) [inline seedlet/classes/class-seedlet-custom-colors.php:298]: The specific feedback here was "those variables should be prefixed". 

I'm not exactly sure what variable the reviewer is referring to, but I changed the name of the JS variable here, as well as adding a prefix to the `$handle`.

File: seedlet/classes/class-seedlet-custom-colors.php [hunk 2]
```diff
@@ -293,7 +292,7 @@ function on_customize_controls_enqueue_scripts() {
 		];
 
 		wp_enqueue_script( $handle, $src, $deps );
-		wp_script_add_data( $handle, 'data', sprintf( 'var _validateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
+		wp_script_add_data( $handle, 'data', sprintf( 'var seedletValidateWCAGColorContrastExports = %s;', wp_json_encode( $exports ) ) );
```
Comment (jffng) [inline seedlet/classes/class-seedlet-custom-colors.php:295]: The specific feedback here was "those variables should be prefixed". 

I'm not exactly sure what variable the reviewer is referring to, but I changed the name of the JS variable here, as well as adding a prefix to the `$handle`.

File: seedlet/template-parts/post/author-bio.php [hunk 3]
```diff
@@ -2,16 +2,14 @@
...
+ * @package Seedlet
  * @since 1.0.0
  */
 
 if ( (bool) get_the_author_meta( 'description' ) ) : ?>
 <div class="author-bio">
 	<?php
-	_e(__( 'Published by', 'seedlet' ));
-	?>
+	_e(__( 'Published by', 'seedlet' )); ?>
```
Comment (jffng) [inline seedlet/template-parts/post/author-bio.php:12]: The reviewer cited this line as a problem, but I'm not sure why.

Comment (kjellr) [reply seedlet/template-parts/post/author-bio.php:12]: Looks like we're unnecessarily using two translation functions there. `__()` will return the translated string, while `_e()` will echo it. In this case, we should just go with `_e()`.

**Comment by jffng** [comment]
Fixed one more bug that came up in the theme sniffer:

```
--------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------------------------------
 44 | ERROR | Overriding WordPress globals is prohibited. Found assignment to $menu
    |       | (WordPress.WP.GlobalVariablesOverride.Prohibited)
--------------------------------------------------------------------------------------------------------------
```

File: seedlet/classes/class-seedlet-custom-colors.php (hunk 2 above)
Comment (jffng) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: @kjellr does this change address the feedback?

Comment (kjellr) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: I _think_ so, but cc @jeffikus or @danieldudzic for another set of eyes. I'm not 100% sure what they're referring to here either.

Comment (danieldudzic) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: No idea what they mean 😅

If they could provide an example, that would be helpful.

Comment (kjellr) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: I'll ask for clarification.

Comment (jeffikus) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: WP Code standards are full word, camel casing with lowercase first letter. This meets that standard in my assessment.

Comment (kjellr) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: @jffng I got some [clarification](https://themes.trac.wordpress.org/ticket/85664#comment:8). We need to update the `_validateWCAGColorContrastExports` variable name, as well as the `validateContrastText` object name. I'm not sure that the `$handle` needs a prefix, but it seems better to give it one.

Comment (jffng) [reply seedlet/classes/class-seedlet-custom-colors.php:295]: Fixed this, mind giving this PR one more look over @kjellr ?

**Pull request merged**